apps/builder/app/builder/features/project-settings/redirect-section.tsx

```diff
@@ -115,7 +118,14 @@ export const RedirectSection = () => {
       return;
     }
 
-    handleSave([{ old: oldPath, new: newPath }, ...redirects]);
+    handleSave([
+      {
+        old: oldPath,
+        new: newPath,
+        status: httpStatus === undefined ? "301" : httpStatus,
```

Comment:
httpStatus ?? "301"

Reply:
I added it first, and then intentionally changed it use `undefined` explicitly 😅. Changing it 👍

Reply:
?? is explicite, it won't treat null or empty string like  ||